lib/rubocop/cop/style/safe_navigation_chain_length.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Enforces safe navigation chains length to not exceed the configured maximum.
```

Comment:
You might want to add a bit of rationale here as well, as one of the most common feedback we receive from users is that they don't understand from the cop docs alone what's the reasoning behind some of our rules.